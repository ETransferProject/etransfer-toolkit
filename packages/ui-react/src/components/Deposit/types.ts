import { TTokenOptionItem, TNetworkItem, TDepositInfo, TToTokenItem } from '@etransfer/types';
import { IChainMenuItem } from '../../types/chain';
import { ComponentStyle } from '../../types/common';
import { ChainId } from '@portkey/types';

export interface DepositSelectGroupProps {
  className?: string;
  depositTokenList: TTokenOptionItem[];
  depositTokenSelected?: TTokenOptionItem;
  depositTokenSelectCallback: (item: TTokenOptionItem) => void;
  networkList: TNetworkItem[];
  networkSelected?: Partial<TNetworkItem>;
  isShowNetworkLoading: boolean;
  networkSelectCallback: (item: TNetworkItem) => Promise<void>;
  chainList: IChainMenuItem[];
  chainSelected?: IChainMenuItem;
  chainChanged: (item: IChainMenuItem) => void;
  receiveTokenList: TTokenOptionItem[];
  receiveTokenSelected?: TTokenOptionItem;
  receiveTokenSelectCallback: (item: TTokenOptionItem) => void;
}

export interface DepositProps {
  containerClassName?: string;
  className?: string;
  componentStyle?: ComponentStyle;
  isShowErrorTip?: boolean;
}

export interface DepositDetailProps {
  className?: string;
  componentStyle?: ComponentStyle;
  isShowErrorTip?: boolean;
  chainItem?: IChainMenuItem;
  depositTokenSymbol: string;
  depositTokenDecimals: number;
  receiveTokenSymbol: string;
  depositInfo: TDepositInfo;
  contractAddress: string;
  contractAddressLink: string;
  qrCodeValue: string;
  tokenLogoUrl?: string;
  showRetry?: boolean;
  onRetry?: () => void;
}

export interface DepositDetailForMobileProps extends DepositDetailProps {
  networkItem?: Partial<TNetworkItem>;
}

export type TDepositForMobileProps = DepositSelectGroupProps &
  Omit<
    DepositDetailForMobileProps,
    'chainItem' | 'depositTokenSymbol' | 'depositTokenDecimals' | 'receiveTokenSymbol' | 'networkItem'
  >;

export type TDepositForWebProps = DepositSelectGroupProps &
  Omit<DepositDetailProps, 'chainItem' | 'depositTokenSymbol' | 'depositTokenDecimals' | 'receiveTokenSymbol'>;

export type TGetNetworkData = {
  chainId: ChainId;
  symbol?: string;
  toSymbol?: string;
};

export interface DepositTokenOptionItem extends TTokenOptionItem {
  toTokenList?: Array<DepositReceiveTokenItem>;
}

export interface DepositReceiveTokenItem extends TToTokenItem {
  chainList?: IChainMenuItem[];
}
