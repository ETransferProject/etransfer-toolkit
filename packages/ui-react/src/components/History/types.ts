import {
  BusinessType,
  OrderStatusEnum,
  RecordsRequestStatus,
  RecordsRequestType,
  TRecordsListItem,
} from '@etransfer/types';
import { ChainId } from '@portkey/types';
import { Moment } from 'moment';

export type THistoryItem = {
  key: string;
  orderType: BusinessType;
  status: OrderStatusEnum;
  arrivalTime: number;
  symbol: string;
  sendingAmount: string;
  receivingAmount: string;
  fromNetwork: string;
  fromAddress: string;
  fromToAddress: string;
  fromChainId: ChainId;
  fromTxId: string;
  toSymbol: string;
  toNetwork: string;
  toFromAddress: string;
  toAddress: string;
  toChainId: ChainId;
  toTxId: string;
  feeInfo: THistoryFeeInfo[];
};

export type THistoryFeeInfo = {
  symbol: string;
  amount: string;
};

export interface HistoryContentProps {
  className?: string;
  recordsList: TRecordsListItem[];
  hasMore: boolean;
  maxResultCount: number;
  totalCount: number;
  skipCount: number;
  onClickItem?: (id: string) => void;
}

export interface HistoryMobileContentProps extends HistoryContentProps {
  onNextPage: () => void;
}

export interface HistoryWebContentProps extends HistoryContentProps {
  onTableChange: (page: number, pageSize: number) => void;
}

export interface HistoryFilterProps {
  className?: string;
  type: RecordsRequestType;
  status: RecordsRequestStatus;
  timestamp: number[] | null;

  onReset: () => void;
}

export interface HistoryMobileFilterProps extends HistoryFilterProps {
  onCloseItem: (type: HistoryCloseItemParams) => void;
  onApply: (params: HistoryFilterOnApplyParams) => void;
}

export interface HistoryWebFilterProps extends HistoryFilterProps {
  onTypeChange: (type: RecordsRequestType) => void;
  onStatusChange: (status: RecordsRequestStatus) => void;
  onTimeStampChange: (timeArray: number[] | null) => void;
}

export type HistoryCloseItemParams = 'type' | 'status' | 'timestamp';

export type HistoryFilterOnApplyParams = {
  type: RecordsRequestType;
  status: RecordsRequestStatus;
  timeArray: number[] | null;
};

export type TRangeValue = [Moment | null, Moment | null] | null;
