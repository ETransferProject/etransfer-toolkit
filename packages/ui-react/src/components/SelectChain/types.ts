import { IChainMenuItem } from '../../types/chain';
import { ComponentStyle } from '../../types/common';
import { TDynamicArrowSize } from '../DynamicArrow';

type TSelectChainCommon = {
  menuItems: IChainMenuItem[];
  selectedItem?: IChainMenuItem;
  getContainer: string;
  className?: string;
  childrenClassName?: string;
  overlayClassName?: string;
  isBorder?: boolean;
  suffixArrowSize?: TDynamicArrowSize;
  hideDownArrow?: boolean;
};

export interface DeviceSelectChainProps extends TSelectChainCommon {
  onClick: (item: IChainMenuItem) => Promise<void> | void;
}

export interface SelectChainProps extends TSelectChainCommon {
  title: string;
  componentStyle?: ComponentStyle;
  clickCallback: (item: IChainMenuItem) => void;
}
