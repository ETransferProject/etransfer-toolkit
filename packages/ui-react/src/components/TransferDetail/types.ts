import { BusinessType, OrderStatusEnum } from '@etransfer/types';
import { ChainId } from '@portkey/types';
import { THistoryFeeInfo } from '../History/types';
import { TransferStatusType } from '../../constants/transfer';

export type TTransferDetailBodyData = {
  id: string;
  orderType: BusinessType;
  status: OrderStatusEnum;
  createTime: number;
  fromNetwork: string;
  fromChainId?: ChainId;
  fromSymbol: string;
  fromIcon?: string;
  fromAddress: string;
  fromToAddress: string;
  fromAmount: string;
  fromAmountUsd: string;
  fromTxId: string;
  fromStatus: OrderStatusEnum | TransferStatusType;
  toNetwork: string;
  toChainId?: ChainId;
  toSymbol: string;
  toIcon?: string;
  toAddress: string;
  toFromAddress: string;
  toAmount: string;
  toAmountUsd: string;
  toTxId: string;
  toStatus: OrderStatusEnum | TransferStatusType;
  toFeeInfo?: THistoryFeeInfo[];
};

export type TTransferDetailActionData = {
  orderId: string;
};
