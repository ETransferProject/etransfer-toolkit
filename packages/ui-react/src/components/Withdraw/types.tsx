import { TNetworkItem, TTokenItem, TWithdrawInfo } from '@etransfer/types';
import { ComponentStyle, IChainMenuItem } from '../../types';
import { FormInstance } from 'antd';

export interface WithdrawProps {
  className?: string;
  chainClassName?: string;
  fromClassName?: string;
  componentStyle?: ComponentStyle;
  isShowMobilePoweredBy?: boolean;
  isShowErrorTip?: boolean;
  isListenNoticeAuto?: boolean;
  depositProcessingCount?: number;
  isShowProcessingTip?: boolean;
  onClickProcessingTip?: () => void;
  onActionChange?: (data: TWithdrawActionData) => void;
  onLogin?: () => void;
}

export type TWithdrawActionData = {
  symbolSelected: string;
  chainSelected: IChainMenuItem['key'];
  addressInput?: string;
  networkSelected?: string;
  processingTransactionCount?: number;
};

export interface WithdrawFormProps {
  className?: string;
  form: FormInstance<TWithdrawFormValues>;
  formValidateData: TWithdrawFormValidateData;
  address?: string;
  minAmount?: string;
  amount?: string;
  receiveAmount?: string;
  balance?: string;
  withdrawInfo: TWithdrawInfo;
  isShowNetworkLoading?: boolean;
  isNetworkDisable?: boolean;
  isSubmitDisabled?: boolean;
  isBalanceLoading?: boolean;
  isTransactionFeeLoading?: boolean;
  componentStyle?: ComponentStyle;
  isShowMobilePoweredBy?: boolean;
  onTokenChange: (item: TTokenItem) => Promise<void>;
  onAddressChange?: (value: string | null) => void;
  onAddressBlur: () => Promise<void>;
  onNetworkChange: (item: TNetworkItem) => Promise<void>;
  onCommentChange?: (value: string | null) => void;
  onClickMax: () => void;
  onAmountChange: React.ChangeEventHandler<HTMLInputElement>;
  onAmountBlur: React.ChangeEventHandler<HTMLInputElement>;
  onClickFailedOk: () => void;
  onClickSuccessOk: () => void;
  onLogin?: () => void;
}

export type TWithdrawFormValidateData = {
  [key in WithdrawFormKeys]: { validateStatus: WithdrawValidateStatus; errorMessage: string };
};

export enum WithdrawValidateStatus {
  Error = 'error',
  Warning = 'warning',
  Normal = '',
}

export enum WithdrawFormKeys {
  TOKEN = 'token',
  ADDRESS = 'address',
  NETWORK = 'network',
  AMOUNT = 'amount',
  COMMENT = 'comment',
}

export type TWithdrawFormValues = {
  [WithdrawFormKeys.TOKEN]: string;
  [WithdrawFormKeys.ADDRESS]: string;
  [WithdrawFormKeys.NETWORK]: TNetworkItem;
  [WithdrawFormKeys.AMOUNT]: string;
};
