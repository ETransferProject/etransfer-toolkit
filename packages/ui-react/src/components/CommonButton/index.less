@import '../../assets/theme/constants.less';

.etransfer-ui-common-button {
  border-radius: 6px;
  box-shadow: none;
  span {
    .font-medium();
  }
}

.etransfer-ui-common-button-primary {
  background: var(--etransfer-ui-brand);
  border: 1px solid var(--etransfer-ui-brand);
  color: var(--etransfer-ui-white-text);
  &:hover {
    background: var(--etransfer-ui-hover);
    border-color: var(--etransfer-ui-hover);
    color: var(--etransfer-ui-white-text);
  }
  &:checked {
    background: var(--etransfer-ui-click);
    border-color: var(--etransfer-ui-click);
    color: var(--etransfer-ui-white-text);
  }
  &:disabled,
  &[disabled]:hover {
    background: var(--etransfer-ui-disable);
    border-color: var(--etransfer-ui-disable);
    color: var(--etransfer-ui-white-text);
  }
  &:focus {
    background: var(--etransfer-ui-click);
    border-color: var(--etransfer-ui-click);
    color: var(--etransfer-ui-white-text);
  }
}

.etransfer-ui-common-button-secondary {
  background: var(--etransfer-ui-normal-bg);
  border: 1px solid var(--etransfer-ui-normal-bg);
  color: var(--etransfer-ui-primary-text);
  &:hover {
    background: var(--etransfer-ui-hover-bg);
    border-color: var(--etransfer-ui-hover-bg);
    color: var(--etransfer-ui-primary-text);
  }
  &:checked {
    background: var(--etransfer-ui-click-bg);
    border-color: var(--etransfer-ui-click-bg);
    color: var(--etransfer-ui-primary-text);
  }
  &:disabled {
    background: var(--etransfer-ui-hover-bg);
    border-color: var(--etransfer-ui-hover-bg);
    color: var(--etransfer-ui-primary-text);
  }
  &:focus {
    background: var(--etransfer-ui-click-bg);
    border-color: var(--etransfer-ui-click-bg);
    color: var(--etransfer-ui-primary-text);
  }
}

.etransfer-ui-common-button-small {
  height: 40px;
  padding: 0 20px;
  span {
    font-size: 14px;
    line-height: 22px;
  }
}

.etransfer-ui-common-button-middle {
  height: 48px;
  padding: 0 28px;
  span {
    font-size: 16px;
    line-height: 24px;
  }
}

.etransfer-ui-common-button-large {
  height: 56px;
  padding: 0 32px;
  span {
    font-size: 16px;
    line-height: 24px;
  }
}

.etransfer-ui-common-button-stretched {
  width: 100%;
}
