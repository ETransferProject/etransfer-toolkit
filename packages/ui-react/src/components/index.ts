// common
export { default as CircleLoading } from './CircleLoading';
export { default as CommonAddress } from './CommonAddress';
export { default as CommonButton } from './CommonButton';
export { default as CommonDrawer } from './CommonDrawer';
export { default as CommonDropdown } from './CommonDropdown';
export { default as CommonImage } from './CommonImage';
export { default as CommonLink } from './CommonLink';
export { default as CommonModal } from './CommonModal';
export { default as CommonModalAutoScreen } from './CommonModalAutoScreen';
export { default as CommonQRCode } from './CommonQRCode';
export { default as CommonSpace } from './CommonSpace';
export { default as CommonSvg } from './CommonSvg';
export { default as CommonModalTip } from './CommonTips/CommonModalTip';
export { default as CommonWarningTip } from './CommonTips/CommonWarningTip';
export { default as ProcessingTip } from './CommonTips/ProcessingTip';
export { default as CommonTooltip } from './CommonTooltip';
export { default as Copy } from './Copy';
export { default as DynamicArrow } from './DynamicArrow';
export { default as EmptyData } from './EmptyData';
export { default as SingleMessage } from './SingleMessage';
export { default as FormAmountInput } from './Form/FormAmountInput';
export { default as FormTextarea } from './Form/FormTextarea';
export { default as GlobalLoading } from './GlobalLoading';
export { default as LinkForBlank } from './LinkForBlank';
export { default as PartialLoading } from './PartialLoading';
export { default as OpenLink } from './OpenLink';
export { default as Header } from './Header';
export { default as MobileSecondPageHeader } from './Header/MobileSecondPageHeader';

// modal
export { default as SimpleTipAutoScreen } from './Modal/SimpleTipAutoScreen';
export { default as SimpleTipModal } from './Modal/SimpleTipModal';
export { default as SynchronizingChainModal } from './Modal/SynchronizingChainModal';
export { default as ViewContractAddressModal } from './Modal/ViewContractAddressModal';
export { default as GoogleReCaptchaIframeModal } from './GoogleReCaptcha/GoogleReCaptchaIframeModal';

// feature - select chain
export { default as SelectChain } from './SelectChain/ChainSelect';
export { default as DepositSelectChain } from './SelectChain/DepositSelectChain';

// feature - select network
export { NetworkLogo } from './NetworkLogo';
export { DepositNetworkResultForMobile, DepositNetworkResultForWeb } from './SelectNetwork/DepositNetworkResult';
export { DepositSelectNetworkForMobile, DepositSelectNetworkForWeb } from './SelectNetwork/DepositSelectNetwork';
// export { NetworkCardForMobile, NetworkCardForWeb } from './SelectNetwork/NetworkCard';
// export { NetworkListSkeletonForMobile, NetworkListSkeletonForWeb } from './SelectNetwork/NetworkListSkeleton';
export { NetworkSelectForMobile, NetworkSelectForWeb } from './SelectNetwork/NetworkSelect';
export { default as NetworkSelectDrawer } from './SelectNetwork/NetworkSelectDrawer';
export { default as NetworkSelectDropdown } from './SelectNetwork/NetworkSelectDropdown';

// feature - select token
export { DepositSelectTokenForMobile, DepositSelectTokenForWeb } from './SelectToken/DepositSelectToken';
// export { TokenCardForMobile, TokenCardForWeb } from './SelectToken/TokenCard';
// export { default as TokenImage } from './SelectToken/TokenImage';
export { TokenSelectForMobile, TokenSelectForWeb } from './SelectToken/TokenSelect';
export { default as TokenSelectDrawer } from './SelectToken/TokenSelectDrawer';
export { default as TokenSelectDropdown } from './SelectToken/TokenSelectDropdown';

// feature module - deposit
export { default as Deposit } from './Deposit';
export { default as Calculator } from './Deposit/Calculator';
export { default as DepositDetailForMobile } from './Deposit/DepositDetailForMobile';
export { default as DepositDetailForWeb } from './Deposit/DepositDetailForWeb';
export { default as DepositForMobile } from './Deposit/DepositForMobile';
export { default as DepositForWeb } from './Deposit/DepositForWeb';
export { default as DepositSelectGroupForMobile } from './Deposit/DepositSelectGroupForMobile';
export { default as DepositSelectGroupForWeb } from './Deposit/DepositSelectGroupForWeb';
export { default as ExchangeRate } from './Deposit/ExchangeRate';

// feature module - withdraw
export { default as Withdraw } from './Withdraw';
export { default as WithdrawForm } from './Withdraw/WithdrawForm';
export { default as WithdrawFooter } from './Withdraw/WithdrawFooter';

// feature module - history
export { default as History } from './History';
export { default as HistoryWebFilter } from './History/HistoryWebFilter';
export { default as HistoryMobileFilter } from './History/HistoryMobileFilter';
export { default as HistoryWebTable } from './History/HistoryWebTable';
export { default as HistoryMobileInfiniteList } from './History/HistoryMobileInfiniteList';
export { default as AddressBox } from './History/AddressBox';
export { default as AmountBox } from './History/AmountBox';
export { default as ArrivalTimeBox } from './History/ArrivalTimeBox';
export { default as FeeInfo } from './History/FeeInfo';
export { default as FromAndToBox } from './History/FromAndToBox';

// feature module - TransferDetail
export { default as TransferDetail } from './TransferDetail';
export { default as MobileTransferDetail } from './TransferDetail/MobileTransferDetail';
export { default as WebTransferDetail } from './TransferDetail/WebTransferDetail';
export { default as TransferDetailBody } from './TransferDetail/TransferDetailBody';
export { default as TransferDetailMain } from './TransferDetail/TransferDetailMain';
export { default as TransferDetailStep } from './TransferDetail/TransferDetailStep';
export { default as FromOrToChain } from './TransferDetail/FromOrToChain';
export { default as TokenAmount } from './TransferDetail/TokenAmount';
export { default as TransferStatus } from './TransferDetail/TransferStatus';
export { default as WalletAddress } from './TransferDetail/WalletAddress';

// feature module - ETransferMain
export { default as ETransferContent } from './ETransferContent';
