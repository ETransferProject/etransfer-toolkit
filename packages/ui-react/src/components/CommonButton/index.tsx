import { Button, ButtonProps } from 'antd';
import clsx from 'clsx';
import './index.less';
import { CommonButtonSize, CommonButtonType } from '../../types/components';

export type CommonButtonProps = Omit<ButtonProps, 'size' | 'type'> & {
  size?: CommonButtonSize;
  type?: CommonButtonType;
  stretched?: boolean;
};

export default function CommonButton({
  size = CommonButtonSize.Middle,
  type = CommonButtonType.Primary,
  stretched = false,
  ...props
}: CommonButtonProps) {
  return (
    <Button
      {...props}
      className={clsx(
        'etransfer-ui-common-button',
        `etransfer-ui-common-button-${size}`,
        `etransfer-ui-common-button-${type}`,
        { ['etransfer-ui-common-button-stretched']: stretched },
        props.className,
      )}
    />
  );
}
