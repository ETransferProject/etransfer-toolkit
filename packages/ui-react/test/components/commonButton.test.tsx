import { beforeEach, describe, expect, test, jest } from '@jest/globals';
import { render, screen } from '@testing-library/react';
import CommonButton, { CommonButtonProps } from '../../src/components/CommonButton';
import { CommonButtonSize, CommonButtonType } from '../../src/types/components';
import clsx from 'clsx';

// Mock Antd Button
jest.mock('antd', () => ({
  Button: jest.fn(({ className, ...props }) => <button {...props} className={className} data-testid="antd-button" />),
}));

// Emulate clsx class name generation
jest.mock('clsx', () => jest.fn(() => 'mocked-classname'));

describe('CommonButton', () => {
  const defaultProps: CommonButtonProps = {
    size: CommonButtonSize.Middle,
    type: CommonButtonType.Primary,
    stretched: false,
  };

  beforeEach(() => {
    (clsx as jest.Mock).mockClear();
  });

  test('Correct rendering of default configuration', () => {
    render(<CommonButton>Test Button</CommonButton>);

    // Assert that the button is rendered
    const button = screen.getByTestId('antd-button');

    // Assert that the button text is correct
    expect(button.textContent).toBe('Test Button');
  });

  describe('Class name generation logic', () => {
    const classCases: Array<{
      props: Partial<CommonButtonProps>;
      expectedClasses: string[];
    }> = [
      {
        props: { size: CommonButtonSize.Small },
        expectedClasses: ['etransfer-ui-common-button-Small'],
      },
      {
        props: { type: CommonButtonType.Secondary },
        expectedClasses: ['etransfer-ui-common-button-Secondary'],
      },
      {
        props: { stretched: true },
        expectedClasses: ['etransfer-ui-common-button-stretched'],
      },
      {
        props: { className: 'custom-class' },
        expectedClasses: ['custom-class'],
      },
    ];

    test.each(classCases)('Generates the correct class name when passing $props', ({ props }) => {
      render(
        <CommonButton {...defaultProps} {...props}>
          Button
        </CommonButton>,
      );

      expect(clsx).toHaveBeenCalled();
    });

    test('Merge multiple class names', () => {
      render(
        <CommonButton size={CommonButtonSize.Large} type={CommonButtonType.Secondary} stretched className="extra-class">
          Button
        </CommonButton>,
      );

      // Assert that the correct class names are generated
      expect(clsx).toHaveBeenCalledWith(
        'etransfer-ui-common-button',
        'etransfer-ui-common-button-large',
        'etransfer-ui-common-button-secondary',
        { ['etransfer-ui-common-button-stretched']: true },
        'extra-class',
      );
    });
  });

  test('Handling undefined and null values', () => {
    render(
      <CommonButton size={undefined} type={undefined} stretched={undefined}>
        Button
      </CommonButton>,
    );

    // Assert that the correct class names are generated
    expect(clsx).toHaveBeenCalledWith(
      'etransfer-ui-common-button',
      'etransfer-ui-common-button-middle',
      'etransfer-ui-common-button-primary',
      { ['etransfer-ui-common-button-stretched']: false },
      undefined,
    );
  });

  test('Snapshot Matching', () => {
    const { asFragment } = render(
      <CommonButton size={CommonButtonSize.Small} type={CommonButtonType.Primary} stretched className="custom-class">
        Submit
      </CommonButton>,
    );

    // Assert that the component matches the snapshot
    expect(asFragment()).toMatchSnapshot();
  });
});
