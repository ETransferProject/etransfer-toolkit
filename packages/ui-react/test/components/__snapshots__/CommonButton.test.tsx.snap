// Jest Snapshot v1, https://goo.gl/fbAQLP
exports[`CommonButton Component Snapshot Matching 1`] = `
<DocumentFragment>
  <button
    class="mocked-classname"
    data-testid="antd-button"
  >
    Submit
  </button>
</DocumentFragment>
`;
